extensions/BMO/template/en/default/account/create.html.tmpl

```diff
@@ -185,5 +210,4 @@ function onSubmit() {
     <div>[% desc FILTER html %]</div>
   </td>
   </tr>
-[% END %]
-
+[% END %e]
```

Comment:
I'm dependent on someone with a working build at the moment. Going to try Dylan's vagrant branch.